### ISSUE TITLE
Test failed 7/180: Exception: SegFault

### ISSUE THREAD

**Issue by Hellseher** [litespeedtech/ls-qpack#51]
Hi,

During preparing your project for Guix downstream package I've faced with unit test failure:
```
 7/180 Test   #7: dyn_table_cap_mismatch ..................***Exception: SegFault  0.00 sec
qdec: debug: initialized.  max capacity=16384; max risked streams=100
```
I did not try to patch it it and just invoke cmake build with `-DLSQPACK_TESTS=ON` flag.

ls-qpack: v2.5.3

**Comment by bkmgit** [comment]
Same problem on Fedora 40
https://download.copr.fedorainfracloud.org/results/fed500/lsqpack/fedora-rawhide-x86_64/06555163-ls-qpack/builder-live.log.gz

**Comment by MatthewGentoo** [comment]
The test is failing because `fopen("testdata/encoder_stream", "r");` fails. It depends on what your working directory is when you invoke the tests.

**Comment by litespeedtech** [comment]
fixed in 2.5.4 .
